src/sonic-py-common/sonic_py_common/logger.py

```diff
@@ -0,0 +1,56 @@
+import os
+import syslog
+
+"""
+Logging functionality for SONiC Python applications
+"""
+
+FACILITY_USER = syslog.LOG_USER
```

Comment:
IMO there is an issue in current logger implementation, there is no log level control, by default all the debug log will be printed out, if we can have some way to control the log level it would be great.

Reply:
Thanks for bringing attention to this. I can add that to this PR fairly easily. What should we set as the default maximum log level -- NOTICE? INFO?

Reply:
I added a new commit which adds a new method, `set_min_log_priority()`, and sets the default minimum priority to NOTICE. We can change the default to INFO if there's a consensus.

Reply:
This is Great Joe.